release/models/bgp/openconfig-bgp-types.yang

```diff
@@ -579,6 +585,48 @@ module openconfig-bgp-types {
                 '429[0-3][0-9]{6}|42[0-8][0-9]{7}|4[0-1][0-9]{8}|[1-3][0-9]{9}|'  +
                 '[1-9][0-9]{1,8}|[0-9])$';
       }
+      type string {
```

Comment:
This looks to be an unrelated change to the description. Incorrect files added in this branch perhaps?